### PR TITLE
14 creat classes module in the back end

### LINKED ISSUE #14: Creat Classes Module in the back-end.
Classes module must be for:
* Create an endpoint to return a map for model classes and UI Classes.
* Create an endpoint to update UI Class for a specific model class.

### PR DESCRIPTION
**Pull request by iyassalah**
@yamenw this is done and works, but please do the changes you talked about.

### CODE REVIEW

File: app/routers/classes.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        internal_label (str): The internal label to be updated.
+        target_lable (str): The new target label.
+
+    Returns:
+        dict: A dictionary containing a message indicating whether the target label was updated successfully or not.
+
+    """
+    query = {"internal_label": internal_label}
+    update = {"$set": {"target_lable": target_lable}}
+    result = classes_collection.update_one(query, update)
```
Comment (yamenw) [inline app/routers/classes.py]: Move all DB CRUD operations to `crud.py` for the sake of abstraction.

File: app/routers/classes.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+def store_classes():
+    """
+    Stores all the classes from `get_classes()` into the database at startup.
+
+    """
+    classes = get_classes()
+    for model_class in classes:
+        query = {"modelClass": model_class}
+        existing_class = mappings_col.find_one(query)
```
Comment (yamenw) [inline app/routers/classes.py]: Bad idea, you have nested loops here, what you should do instead here is:
1. Create a unique index on the internal names
2. Try inserting a new entry 
3. Catch duplicate key errors and ignore them, also consider moving this function to `crud.py`

**Review by yamenw — APPROVED** [review]
Just merge it already

File: app/routers/classes.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    query = {"internalName": internal_name}
+    update = {"$set": {"displayedName": displayed_name}}
+    result = mappings_col.update_one(query, update)
+    if result.modified_count == 1:
+        return {"message": "Target label updated successfully"}
+    else:
+        return {"message": "No documents were modified"}
+
+
+# Return all tags from the database for the UI
```
Comment (yamenw) [inline app/routers/classes.py]: Move comments to docstrings

File: app/routers/classes.py [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+@router.get("/classes/")
+async def get_uiclasses():
+    uiclasses = []
+    for doc in mappings_col.find({}, {"_id": 0, "displayedName": 1}):
```
Comment (yamenw) [inline app/routers/classes.py]: Don't just returns the display names in an array, how will we know which display name belongs to which internal name? I don't think its sorted so we can't just use order, return both of them in a dictionary/JSON.

File: app/routers/classes.py (hunk 2 above)
Comment (iyassalah) [reply app/routers/classes.py]: got it thanks alot.

File: app/routers/classes.py (hunk 3 above)
Comment (iyassalah) [reply app/routers/classes.py]: done

File: app/routers/classes.py (hunk 4 above)
Comment (iyassalah) [reply app/routers/classes.py]: done

**Pull request merged**